src/main/app/src/components/NavBar.js

```diff
@@ -121,6 +124,54 @@ const ContentSwitch = () => {
   );
 };
 
+const LogOutButton = () => {
```

Comment:
Function is good, but can we refactor this code into its own file?

Like make a `LogoutComponent.js` file and import it here - just to keep the code clean and the components nice and modular.

Reply:
If you do the above refactoring, there are also multiple components in `Inventory.js` that need to be similarly refactored.